pandas/core/strings.py

```diff
@@ -484,8 +493,9 @@ def str_get_dummies(arr, sep='|'):
     1  0  0  0
     2  1  0  1
 
-    See also ``pd.get_dummies``.
-
+    See Also
+    --------
+    ``pd.get_dummies``.
```

Comment:
this should be ``pandas.get_dummies`` (the actual object as how it is called, then a link will be automatically generated)